src/index.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+source "$(dirname "${BASH_SOURCE[0]}")/git_exists.sh"
+
+if ! git_exists "https://$INPUTS_TOKEN@$INPUTS_REPOSITORY.wiki.git" &>/dev/null; then
+  echo "$(basename "$0"): https://$INPUTS_REPOSITORY.wiki.git doesn't exist." >&2
+  echo 'Did you remember to create the first wiki page manually?' >&2
+  echo 'You can find more information on the readme. [1]' >&2
+  echo
+  echo '[1]: https://github.com/spenserblack/actions-wiki#usage'
```

Comment:
This makes me wonder if there's a built-in context or environment variable to get the URL of the action being used.